### PR TITLE
Enigmail doesn't require interfering with the config anymore #346

### PR DESCRIPTION
**Pull request by lefherz**
addresses #346 

@pbrunschwig fine?

### CODE REVIEW

File: doc/install-autocrypt-enigmail.rst [hunk 1]
```diff
@@ -16,7 +16,6 @@ You can test Autocrypt in Enigmail already today:
 
 - The following setup may require a restart of Thunderbird
 
-- Then, navigate to "Edit / Preferences / Advanced / Config Editor", read the warning and 
-  scroll down to "extensions.enigmail.autocryptMode": Double-click the entry and enter a `1`
+- Autocrypt is now running. You can configure it at: Account Manager > OpenPGP Security.
```
Comment (hpk42) [inline doc/install-autocrypt-enigmail.rst]: "is now running" sounds odd.  Shouldn't it just say "You can enable autocrypt here: Account Manager > OpenPGP Security" and maybe even mention the precise option? (Haven't installed the latest nightly yet so can't say what it is)

Comment (azul) [reply doc/install-autocrypt-enigmail.rst]: It's more disable, right?
How about "Autocrypt is part of the default configuration. You can configure and disable it at:..." (If one can also configure it...)
Will install latest nightly tomorrow and try.

Comment (pbrunschwig) [reply doc/install-autocrypt-enigmail.rst]: I agree with @azul, that sounds better.

Comment (lefherz) [reply doc/install-autocrypt-enigmail.rst]: I'm fine with "Autocrypt is part of the default configuration", but we should remove the bullet point - it isn't part of the step-by-step guide anymore.

**Pull request merged**